chapter05/spg/exercise1.txt

```diff
@@ -0,0 +1 @@
+The code is trying to define a hash using the syntax for creating a reference to an anonymous hash. 'my $passenger_1 = { ... };' would be valid.
```

Comment:
Good good.

Alternatively, `tr/{}/()/ ` on the snippet to make them fully fledged hashes, then the final line won't need to be changed.  I suppose it depends on how the data is going to be used later on.